tests/unit/test-util/test.adb.js

```diff
@@ -392,36 +399,41 @@ describe('utils/adb', () => {
          );
        });
 
-    it('does not reject when all the required permissions have been granted',
-       async () => {
-         const adb = getFakeADBKit({
-           adbClient: {
-             shell: sinon.spy(() => Promise.resolve('')),
-           },
-           adbkitUtil: {
-             readAll: sinon.spy(() => {
-               return Promise.resolve(
-                 new Buffer(fakeAndroidGrantedPermissions)
-               );
-             }),
-           },
-         });
-         const adbUtils = new ADBUtils({adb});
+    describe('Android Granted Permissions discovery', async () => {
+      async function assertPermissions(fakeAndroidPmDump) {
+        const adb = getFakeADBKit({
+          adbClient: {
+            shell: sinon.spy(() => Promise.resolve('')),
+          },
+          adbkitUtil: {
+            readAll: sinon.spy(() => {
+              return Promise.resolve(new Buffer(fakeAndroidPmDump));
```

Comment:
`new Buffer(string)` is deprecated. It was already pre-existing, but since you're editing anyway, could you change it to `Buffer.from(fakeAndroidPmDump)` ?

Reply:
yeah, I just added a separate commit that replace it also for other usages of "new Buffer(...)" in the same test file.